### ISSUE TITLE
Support for streaming video with extension on Kiwi Browser on Android

### ISSUE THREAD

**Issue by keineahnungking** [killergerbah/asbplayer#438]
Currently when I install the extension on Kiwi Browser on Android and open the extension, it just opens a settings popup in a new tab but doesn't give me the option to select subtitles.

**Comment by killergerbah** [comment]
Opening the extension from the webpage itself (menu -> extensions -> asbplayer) should open up the subtitle track selector.

**Comment by killergerbah** [comment]
Yeah that's what I mean. I think I understand the issue now, though. Basically asbplayer isn't good at detecting tablets. I have a speculative fix for this.

**Comment by keineahnungking** [comment]
I have this issue with Kiwi Browser on both my phone and my tablet. I don't have this issue with Firefox on Android on both my phone and my tablet. I believe the issue is with Kiwi Browser.

**Comment by killergerbah** [comment]
Interesting
Our of curiosity are you using the desktop version of the website?

**Comment by keineahnungking** [comment]
I tried on YouTube with both mobile and desktop version and both didn't work

**Comment by killergerbah** [comment]
Can you try this build?
[asbplayer-extension-1.3.0-chromium.zip](https://github.com/user-attachments/files/15905565/asbplayer-extension-1.3.0-chromium.zip)

**Comment by keineahnungking** [comment]
The subtitle selector pops up now but I am unable to open the settings and unable to open the dialog to mine a card. The buttons in the overlay don't do anything

**Comment by killergerbah** [comment]
You can open settings from the extensions manager. I am surprised that the buttons in the overlay do nothing, need more info to debug.

**Comment by keineahnungking** [comment]
For some reason the overlay buttons work on a different instance of Kiwi Browser. What is still not working is capturing screenshots. On YouTube I can at least capture the sentence audio. On Emby nothing is captured. Also the Anki dialog is not opened when selecting that option. These are the errors I'm getting:

Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'substring')
Context
background.js
Stack Trace
background.js:2 (anonymous function)

Uncaught (in promise) Error: Failed to start recording
Context
background.js
Stack Trace
background.js:2 (anonymous function)

Error starting tab capture
Context
offscreen-audio-recorder.html
Stack Trace
offscreen-audio-recorder.js:1 (anonymous function)

Unchecked runtime.lastError: No active web contents to capture
Context
Unknown
Stack Trace
:0 (anonymous function)

**Comment by killergerbah** [comment]
Unfortunately screenshots do not work on Kiwi due to lack of screenshot API support. So you will have to live without screenshots until that is fixed on Kiwi.

**Comment by killergerbah** [comment]
All actionable issues have been addressed.
